backend/lcfs/web/api/compliance_report/views.py

```diff
@@ -70,7 +66,13 @@ async def get_compliance_reports(
     pagination.filters.append(
         FilterModel(field="status", filter="Draft", filter_type="text", type="notEqual")
     )
-    return await service.get_compliance_reports_paginated(pagination)
+
+    try:
```

Comment:
The @view_handler should be mapping the DataNotFoundException to a 404 already.
```
            except DataNotFoundException:
                raise HTTPException(status_code=404, detail="Not Found")
```
Can this be removed?

Reply:
Thank you for catching this!